hyfer-backend/server/datalayer/githubSync.js

```diff
@@ -1,58 +1,170 @@
+/* eslint-disable camelcase */
+require('dotenv').config();
+const moment = require('moment');
 const db = require('./database');
```

Comment:
Make sure that the `moment` package is added to `package.json`. Currently it is not.